front/src/components/mainPage/PosterBox.jsx

```diff
@@ -7,7 +7,12 @@ function PosterBox(props) {
     const navigate = useNavigate();
 
     const ticketingBtn = () => {
```

Comment:
```suggestion
    const handleTicketing= () => {
```
핸들러 함수 이름이 이게 더 나을거같아요

Reply:
확인했습니다